packages/react-button/etc/react-button.api.md

```diff
@@ -229,7 +233,7 @@ export interface SplitButtonProps extends ButtonProps, MenuButtonProps {
 export const splitButtonShorthandProps: string[];
 
 // @public (undocumented)
-export interface SplitButtonState extends Omit<SplitButtonProps, 'menu'>, MenuButtonState {
+export interface SplitButtonState extends Omit<SplitButtonProps, 'omit'>, MenuButtonState {

```

Comment:
Omit omit?

Reply:
Oh lol, that should be menu, but maybe it's not even needed if the build is not complaining.